pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -93,31 +98,27 @@ buildPythonPackage rec {
 
   checkInputs = [
     pytestCheckHook
+    black
```

Comment:
Please patch out the black tests. Updating black should not trigger to many rebuilds or cause regressions.

Reply:
They have a whole util method for that now, apparently, which compares decompiled code via pytest-snapshot. So I adjusted the derivation in `postPatch` to do a sed removal for calls to that util method and then to fix up the remaining tests that manually specify `format=True` on `ibis.decompile` (defined in `ibis/tests/util.py` and what specifically uses black)